packages/flutter/lib/src/material/menu_bar.dart

```diff
@@ -0,0 +1,3396 @@
+                  actions: <Type, Action<Intent>>{
+                    DirectionalFocusIntent: _MenuDirectionalFocusAction(controller: _controller),
+                    DismissIntent: _MenuDismissAction(controller: _controller),
+                  },
+                  child: Shortcuts(
+                    shortcuts: _kMenuTraversalShortcuts,
+                    child: _MenuPanel(
+                      menuStyle: widget.style,
+                      clipBehavior: widget.clipBehavior,
+                      statesController: _statesController,
+                      orientation: Axis.horizontal,
+                      children: MenuItemGroup._expandGroups(widget.children, Axis.horizontal),
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  @override
+  List<DiagnosticsNode> debugDescribeChildren() {
+    return <DiagnosticsNode>[...widget.children.map<DiagnosticsNode>((Widget item) => item.toDiagnosticsNode())];
+  }
+}
+
+/// A button for use in a [MenuBar] that can be activated by click or keyboard
+/// navigation that displays a shortcut hint and optional leading/trailing
+/// icons.
+///
+/// This widget represents a leaf entry in a menu that is typically part of a
+/// [MenuBar], but may be used independently, or as part of a menu created with
+/// [createMaterialMenu].
+///
+/// It shows a hint for an associated shortcut, if any. When selected via click,
+/// hitting enter while focused, or activating the associated [shortcut], it
+/// will call its [onPressed] callback. If neither is defined, then this item
+/// will be disabled.
+///
+/// {@macro flutter.material.menu_bar.shortcuts_note}
+///
+/// See also:
+///
+/// * [MenuBar], a class that creates a top level menu bar in a Material Design
+///   style.
+/// * [MenuButton], a menu item which manages a submenu.
+/// * [MenuItemGroup], a menu item which collects its members into a group
+///   separated from other menu items by a divider.
+/// * [MenuItemButton], a leaf menu item which displays the label, an optional
+///   shortcut label, and optional leading and trailing icons.
+/// * [createMaterialMenu], a function that creates a [MenuEntry] that allows
+///   creation and management of a cascading menu anywhere.
+/// * [MenuController], a class that allows controlling and connecting menus.
+/// * [PlatformMenuBar], which creates a menu bar that is rendered by the host
+///   platform instead of by Flutter (on macOS, for example).
+/// * [ShortcutRegistry], a registry of shortcuts that apply for the entire
+///   application.
+/// * [VoidCallbackIntent] to define intents that will call a [VoidCallback] and
+///   work with the [Actions] and [Shortcuts] system.
+/// * [CallbackShortcuts] to define shortcuts that simply call a callback and
+///   don't involve using [Actions].
+/// * [PlatformMenuBar], a class that renders similar menu bar items from a
```

Comment:
`PlatformMenuBar` is already listed above (line 322)

Reply:
Thanks!